tests/unit/test_infos.py

```diff
@@ -0,0 +1,101 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
```

Comment:
perhaps test_project_info.py ?

Reply:
This test file will also contain tests for `PartInfo` and `StepInfo` (which will also be added to `infos.py`).